### DISCUSSION TITLE
How can I permanetly add the following 'echo "2" > /sys/devices/cpu/rdpmc' to RHCOS ?

### DISCUSSION THREAD

**Discussion by midu16** [ostreedev/ostree#3040] [Q&A] [answered]
I would like to be able to permanently add the following echo value `echo "2" > /sys/devices/cpu/rdpmc`to RHCOS through a machine-config in OCPv4.12.X on-wards

**Comment by cgwalters** [comment, marked as answer]
Hi, I know this is a bit confusing but this forum is for ostree, which is low level OS agnostic tooling to update operating systems.  It's independent of OCP for example.

Now, directly writing to `/sys` via Ignition will likely break with MachineConfig; what you'll need to do instead is write a systemd unit that does e.g. `ExecStart=/bin/sh -c 'echo 2 > /sys/devices/cpu/rdpmc'` for example.
